.gitignore

```diff
@@ -21,3 +21,6 @@ yarn.lock
 
 # mkdocs artifact
 site
+
+examples/**/*.js
```

Comment:
This does not hurt, although I am curious why did you have js files under examples and lib? tsconfig is setting `dist` as the output which is already in the gitignore.

Reply:
I think it was due to I compile from another repository, which references this repository with direct path (not packaged), like this:

`import * as ssp from '../../ssp-amazon-eks/lib';`

Let me revert these two lines from `.gitignore` and figure out a better way to reference in dev environment.